nibabel/gifti/gifti.py

```diff
@@ -562,40 +576,31 @@ def to_file_map(self, file_map=None):
         Returns
         -------
         None
-
-        Notes
-        -----
-        We write all files with utf-8 encoding, and specify this at the top of
-        the XML file with the ``encoding`` attribute.
-
-        The Gifti spec suggests using the following suffixes to your
-        filename when saving each specific type of data:
-
-        .gii
-            Generic GIFTI File
-        .coord.gii
-            Coordinates
-        .func.gii
-            Functional
-        .label.gii
-            Labels
-        .rgba.gii
-            RGB or RGBA
-        .shape.gii
-            Shape
-        .surf.gii
-            Surface
-        .tensor.gii
-            Tensors
-        .time.gii
-            Time Series
-        .topo.gii
-            Topology
-
-        The Gifti file is stored in endian convention of the current machine.
         """
-        # Our giftis are always utf-8 encoded - see GiftiImage.to_xml
         if file_map is None:
             file_map = self.file_map
         f = file_map['image'].get_prepare_fileobj('wb')
         f.write(self.to_xml())
+
+    @classmethod
+    def from_file_map(klass, file_map, buffer_size=35000000):
+        """ Load a Gifti image from a file_map
+
+        Parameters
```

Comment:
Not numpy-docstring compliant.

Also - file_map is not a string, but a dictionary, containing fileholders.